idea_town/frontend/static-src/app/templates/not-found-page.js

```diff
@@ -0,0 +1,6 @@
+export default `
+  <section class="page" data-hook="not-found-page">
+    <h1>404 wut</h1>
+  </section>`
```

Comment:
My recollection is that you can only specifically target a single language, but that other (popular?) languages are available for use (just can't force a specific version).